src/bluesearch/database/download.py

```diff
@@ -47,14 +56,31 @@ def get_days_list(
     if end_date is None:
         end_date = datetime.today()
 
-    delta = end_date - start_date
+    date_list = []
+
+    if delta == "day":
+        start_date = datetime(start_date.year, start_date.month, start_date.day)
+        end_date = datetime(end_date.year, end_date.month, end_date.day)
 
-    days_list = []
-    for i in range(delta.days + 1):
-        day = start_date + timedelta(days=i)
-        days_list.append(day)
+        current_date = start_date
+        while current_date <= end_date:
+            date_list.append(current_date)
+            current_date += timedelta(days=1)
 
-    return days_list
+    elif delta == "month":
+        start_date = datetime(start_date.year, start_date.month, 1)
+        end_date = datetime(end_date.year, end_date.month, 1)
+
+        current_date = start_date
+        while current_date <= end_date:
+            date_list.append(current_date)
+            current_date_ = current_date + timedelta(days=32)
+            current_date = current_date_.replace(day=1)
```

Comment:
StackOverflow was also leading us to the `dateutil` package but we wanted to avoid using external packages. However, I did not realize that `pandas` (and others) depend on it so it definitely is a viable option.

You second suggestion is nice too:) 

Which one should we go for then?  Personally, I don't have any preferences
* `timedelta(days=32)`
* `dateutil`
* "divmod"

@Stannislav @EmilieDel

Reply:
I don't mind. I think either way is fine.

Reply:
OK:) Well, to minimize the effort I suggest keeping it the way it is:) Feel free to resolve if you agree